### PR TITLE
Fix navigation menu icon on Android.

### PR DESCRIPTION
**Pull request by beshaya**
Also make it easier to get to the calculator from the blog post on mobile.
Fixes: #175

### CODE REVIEW

File: src/posts/blog/1-ux-redesign/ux-redesign.ts [hunk 1]
```diff
@@ -16,7 +16,7 @@ const image: ImageMeta = {
 
 const content = `Hi everyone!
 
-We’ve been hard at work improving the microCOVID calculator to make it better and easier to use.  Today we’re announcing Version 2 of our calculator.  Much of the interface will feel familiar, but there have been some changes to reduce confusion and improve the user flow.  
+We’ve been hard at work improving the microCOVID [calculator](../../) to make it better and easier to use.  Today we’re announcing Version 2 of our calculator.  Much of the interface will feel familiar, but there have been some changes to reduce confusion and improve the user flow.
```
Comment (dominic) [inline src/posts/blog/1-ux-redesign/ux-redesign.ts:19]: Absolute path would be safer if this page ever changes

**Review by dominic — APPROVED** [review]

File: src/posts/blog/1-ux-redesign/ux-redesign.ts (hunk 1 above)
Comment (blanchardjeremy) [reply src/posts/blog/1-ux-redesign/ux-redesign.ts]: Resolved

**Review by blanchardjeremy — APPROVED** [review]
LGTM

**Comment by blanchardjeremy** [comment]
@beshaya did you mean to request a review after ti's already been merged?

**Pull request merged**